ui/components-react/src/components-react/filter-builder/FilterBuilderState.ts

```diff
@@ -223,7 +220,7 @@ export interface UsePropertyFilterBuilderProps {
  * Options for [[UsePropertyFilterBuilderResult.buildFilter]].
  * @beta
  */
-export interface BuildFilterOptions {
+export interface FilterOptions {
```

Comment:
Keep the previous name but deprecate it, so we can reduce code breaking change to a minimum. (Even if it is unlikey that someone actually imported this by name...)
```suggestion
 * @deprecated in 4.7.0. Use [[FilterOptions]].
 */
export type BuildFilterOptions = FilterOptions;

/** Options for [[UsePropertyFilterBuilderResult.buildFilter]].
 * @beta
 */
export interface FilterOptions {
```

Reply:
Was there are reason for renaming this interface?